Makefile

```diff
@@ -383,12 +383,18 @@ $(FV_TESTS): vendor/.up-to-date $(FELIX_GO_FILES)
 
 .PHONY: fv
 fv: calico/felix bin/iptables-locker bin/test-workload bin/test-connection $(FV_TESTS)
+	# Copy the ginkgo binary out of the container since we need to run the fv tests directly
+	# on the host (because they need to be able to manipulate docker).  It'd be even nicer
+	# if we could give the build container access to the docker API but we've so-far struggled
+	# to get that working.
 	@echo Running Go FVs.
+	mkdir -p bin
```

Comment:
We know that bin already exists, because of the deps in the `fv:` line above.  So this will always be a no-op, and I think it's worth saving a line here.

Alternatively, another approach that would be usable for other future targets, is:
```
bin/ginkgo:
       mkdir -p bin
       $(DOCKER_GO_BUILD) cp /go/bin/ginkgo bin/ginkgo
```
and then add bin/ginkgo as another dep of the `fv:` line.